templates/github/.ci/scripts/docs-builder.py.j2

```diff
@@ -96,7 +96,7 @@ def main():
         r = requests.get("https://pypi.org/pypi/{{ plugin_dash }}/json")
         latest_version = version.parse(json.loads(r.text)["info"]["version"])
         docs_version = version.parse(branch)
-        if latest_version == docs_version:
+        if docs_version >= latest_version:
```

Comment:
Could you add comment please - `This is to mitigate delays on PyPI which doesn't update metadata in timely manner. It doesn't prevent incorrect docs being published at root if 2 releases are done close to each other, within PyPI delay. E.g. Release 3.11.0 an then 3.10.1 immediately after.`
I'm worried that we'll forget all those details.